locale/fr/get-involved/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant 
```

Comment:
'excitée de permettre' -> I'm not a big fan here.
* 'excitée a l'idee de permettre' ?
* something else ?

Reply:
'excitée a l'idee de permettre' => c'est tout much AMA

Reply:
yes, the problem is in the word "excite" actually. It's a too literal translation,

Reply:
Maybe simply "heureuse"?

Reply:
better yes!